zerver/lib/events.py

```diff
@@ -1208,22 +1207,6 @@ def _draft_update_action(i: int) -> None:
                             for user_id in user_group["members"]
                             if user_id != person_user_id
                         ]
-
-                    for setting_name in Realm.REALM_PERMISSION_GROUP_SETTINGS:
```

Comment:
Previously, we send setting update events only for guests who cannot access the user. For all other users, clients were responsible to update the settings on receiving `update` event.

But the setting value did not change previously, now setting value is actually changed as we are removing the deactivated user from group. So, have updated the code to send events for setting updates for all users.